app/views/tag/rss.rss.builder

```diff
@@ -22,10 +25,12 @@ xml.rss :version => "2.0" do
         else
           xml.pubDate     node.created_at.to_s(:rfc822)
         end
-        #xml.link        url_for :only_path => false, :controller => 'notes', :action => 'show', :id => node.nid
         xml.link        "https://" + request.host.to_s + node.path
-        #xml.image "//#{ request.host }/"+node.main_image.path(:default) if node.main_image
-        xml.description auto_link(node.latest.render_body, :sanitize => false)
+        if node.main_image
+          xml.description  {  xml.cdata!("<img src='#{node.main_image.path(:default)}' alt='#{node.main_image.title}'><p>#{auto_link(node.latest.render_body, :sanitize => false)}</p>")  }
+        else
+          xml.description  {  xml.cdata!("<img src='https://i.publiclab.org/system/images/photos/000/000/354/medium/Boots-ground-02.png' alt='PublicLab'><p>#{auto_link(node.latest.render_body, :sanitize => false)}</p>")  }
```

Comment:
Hi , can you add this image [https://publiclab.org/system/images/photos/000/023/444/original/Screenshot_20180204-101546_2.png](https://publiclab.org/system/images/photos/000/023/444/original/Screenshot_20180204-101546_2.png) as used in #2221 to maintain uniformity . Awesome !